index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Fahim's Portfolio</title>
+  </head>
+  <body>
+    <div class="pf-wrap">
+      <div class="pf-header">
+        <a class="pf-logo" href=""><img class="pf-logo" src="../portfolio-website/img/F_logo.png" alt="Fahim logo" /></a>
+        <span class="material-symbols-outlined"> menu </span>
+        <!-- <a class="mb-menu-button" href="#"><img src="../portfolio-website/img/mobile-menu.svg" alt="mobile-menu" /></a> -->
+      </div>
```

Comment:
- Good job implementing the toolbar. But i will like to kindly suggest that you replaced your `<div class="pf-wrap">` with a header tag instead `<header class="pf-wrap">` because typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

- Can you kindly wrap your hamburger menu in a `<nav>` bar instead of a `span`? The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices. 🤗 

- Kindly delete the commented code on line 16 inorder to have a cleaner code base. 👍🏽